cves/kernel/CVE-2020-12654.yml

```diff
@@ -153,10 +170,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Qing Xu discovered that two methods made calls without proper checks. 
```

Comment:
A little confusing because in the description you say, "method is called" leading me to believe the issue only involved one method.